tests/stubs/logs/20220615.log

```diff
@@ -0,0 +1,4 @@
+[ 2022-06-15 10:24:43 ] 
```

Comment:
Please consider adding `.gitignore` file in this directory. I think we do not need to have entire build history in the repository.

Reply:
@krzysztofrewak change applied, please resolve conversation